src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/sentencing/SentencingAdjustmentsToNomisTest.kt

```diff
@@ -680,6 +818,20 @@ class SentencingAdjustmentsToNomisTest : SqsIntegrationTestBase() {
     ).get()
   }
 
+  private fun publishUpdateAdjustmentDomainEvent() {
+    val eventType = "sentencing.sentence.adjustment.updated"
```

Comment:
is "sentence" correct here?

Reply:
Probably not. But all made up right now. No idea what real event will be